commons/protocol/src/test/resource/saved-protocol/0.2/array/seq_int.json

```diff
@@ -0,0 +1 @@
+[1.0,2.0,3.0]
```

Comment:
though your patch doesn't create the problem, afaik, I already had a card about it (in the context of ids)